src/lib/ng-v5/entry-point/ts/compile-ngc.transform.ts

```diff
@@ -20,33 +20,31 @@ export const compileNgcTransform: Transform = transformFromPromise(async graph =
   const { basePath, cssUrl, styleIncludePaths } = entryPoint.data.entryPoint;
   const stylesheetProcessor = new StylesheetProcessor(basePath, cssUrl, styleIncludePaths);
 
-  await Promise.all([
-    compileSourceFiles(
-      graph,
-      entryPoint,
-      tsConfig,
-      moduleResolutionCache,
-      stylesheetProcessor,
-      {
-        outDir: path.dirname(esm2015),
-        declaration: true,
-        target: ts.ScriptTarget.ES2015
-      },
-      path.dirname(declarations)
-    ),
+  await compileSourceFiles(
```

Comment:
Can't these run concurrently anymore?

Reply:
The reason being if they run concurrently error messages are twice. That said, concurrently in most cases is slower since the IO ops are still a limiting factor.